docs/tutorials/quickstart.md

```diff
@@ -1,186 +1,240 @@
+
+To further debug and diagnose cluster problems, use 'kubectl cluster-info dump'.
 ```
 
-#### Deploying with Powershell
+Administrative note: By default, the directory where aks-engine stores cluster configuration (`_output/contoso-apple` above) won't be overwritten as a result of subsequent attempts to deploy a cluster using the same `--dns-prefix`) To re-use the same resource group name repeatedly, include the `--force-overwrite` command line option with your `aks-engine deploy` command. On a related note, include an `--auto-suffix` option to append a randomly generated suffix to the dns-prefix to form the resource group name, for example if your workflow requires a common prefix across multiple cluster deployments. Using the `--auto-suffix` pattern appends a compressed timestamp to ensure a unique cluster name (and thus ensure that each deployment's configuration artifacts will be stored locally under a discrete `_output/<resource-group-name>/` directory).
+
+**Note**: If the cluster is using an existing VNET, please see the [Custom VNET][custom-vnet] feature documentation for additional steps that must be completed after cluster provisioning.
+
+<a href="#the-long-way"></a>
+
+## AKS Engine the Long Way
 
-```powershell
-Add-AzureRmAccount
+This example uses the more traditional method of generating raw ARM templates, which are submitted to Azure using the `az group deployment create` command.
 
-Select-AzureRmSubscription -SubscriptionID <SUBSCRIPTION_ID>
+For this example, we will use the same information as before: the subscription id is `51ac25de-afdg-9201-d923-8d8e8e8e8e8e`, the DNS prefix is `contoso-apple`, and the location is `westus2`.
+
+Before we do anything, we need to log in to Azure:
+
+```console
+$ az login
+Note, we have launched a browser for you to login. For old experience with device code, use "az login --use-device-code"
+You have logged in. Now let us find all the subscriptions to which you have access...
+```
 
-New-AzureRmResourceGroup `
-    -Name <RESOURCE_GROUP_NAME> `
-    -Location <LOCATION>
+We will also need to generate an SSH key. When creating VMs, you will need an SSH RSA key for SSH access. Use the following articles to create your SSH RSA Key:
+
+1. Windows - https://www.digitalocean.com/community/tutorials/how-to-create-ssh-keys-with-putty-to-connect-to-a-vps
+1. Mac and Linux - https://help.github.com/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent/
+
+Next, we'll create a resource group. A resource group is a container that holds related resources for an Azure solution. In Azure, you logically group related resources such as storage accounts, virtual networks, and virtual machines (VMs) to deploy, manage, and maintain them as a single entity. In this case, we want to deploy, manage and maintain the whole Kubernetes cluster as a single entity.
```

Comment:
```suggestion
Next, we'll create a resource group. A resource group is a container (not to be confused with Docker containers.) that holds related resources for an Azure solution. In Azure, you logically group related resources such as storage accounts, virtual networks, and virtual machines (VMs) to deploy, manage, and maintain them as a single entity. In this case, we want to deploy, manage and maintain the whole Kubernetes cluster as a single entity.
```

Reply:
Super nit: Although I am not comfy with this definition, I understand that this is Microsoft's definition of a resource group and I feel this is disambiguation would be useful for a newbie.

Reply:
changed terminology to "logical grouping unit".

Reply:
Perfect!